### PR TITLE
Add mentioned person

### PR DESCRIPTION
**Pull request by flavens**
Property MentionedPerson has been added in the sidebar of the letter template.

### CODE REVIEW

File: src/app/resource/letter/letter.component.ts [hunk 1]
```diff
@@ -245,7 +247,7 @@ export class LetterComponent implements OnDestroy, OnInit {
                                     LetterComponent.collectImagesAndRegionsForResource(resourceSeq.resources[0]);
 
                                     this.resource = resourceSeq.resources[0];
-                                    // console.log('resource ', this.resource);
+                                    console.log('resource ', this.resource);
```
Comment (tobiasschweizer) [inline src/app/resource/letter/letter.component.ts]: Could you comment out this console.log?

**Review by tobiasschweizer — COMMENTED** [review]
Looks good, thanks a lot! Just one small thing (see comment below).

File: src/app/resource/letter/letter.component.ts (hunk 1 above)
Comment (flavens) [reply src/app/resource/letter/letter.component.ts]: yes

**Review by tobiasschweizer — APPROVED** [review]
Merci beaucoup!

**Pull request merged**